### PR TITLE
Test for ZR transceiver TX laser bias current

### PR DESCRIPTION
**Pull request by ahsaanyousaf**

### CODE REVIEW

File: feature/platform/transceiver/zr_laser_bias_current/README.md [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+
+As per CMIS:
+
+Measured Tx laser bias current is represented as a 16-bit unsigned integer with
+the current defined as the 12 full 16-bit value (0 to 65535) with LSB equal to
+2 uA times the multiplier from Byte 01h:160. For a multiplier of 13 1,
+this yields a total measurement range of 0 to 131 mA.
+
+Accuracy must be better than +/-10% of the manufacturer's nominal value over
+specified 15 operating temperature and voltage.
```
Comment (ejbrever) [inline feature/platform/transceiver/zr_laser_bias_current/README.md:16]: can you please clarify this sentence in regards to the "15"? Not sure I followed :)

Comment (ahsaanyousaf) [reply feature/platform/transceiver/zr_laser_bias_current/README.md:16]: This was just supposed to be "Accuracy must be better than +/-10% of the manufacturer's nominal value over
specified operating temperature and voltage."

Updated :)

Comment (ejbrever) [reply feature/platform/transceiver/zr_laser_bias_current/README.md]: Ah! Thanks :)

**Review by ejbrever — APPROVED** [review]

**Review by dplore — CHANGES_REQUESTED** [review]
Minor suggestion on the CMIS link.   

Please also update testregistry.textproto with this test id

**Pull request merged**